migration/idempotent/014-apply-config.sql

```diff
@@ -70,17 +70,32 @@ DECLARE
 BEGIN
     FOR _sql IN
     (
-        -- find tables which belong to the extension so we can mark them to have
+        -- find tables and sequences which belong to the extension so we can mark them to have
```

Comment:
I don't think that I understand why this is in an idempotent file. Doesn't it only need to run once?

Reply:
Ideally, it would only need to run once, however since we are dealing with both the security of the functions and whether or not data gets included in pg_dumps, I felt it was important enough to run after every migration in case something newly added slipped through the cracks.

Reply:
WDYT?